cinema/serializers.py

```diff
@@ -0,0 +1,21 @@
+from rest_framework import serializers
+from cinema.models import Movie
+
+
+class MovieSerializer(serializers.Serializer):
+    id = serializers.IntegerField(read_only=True)
+    title = serializers.CharField(max_length=255, required=False)

```

Comment:
The `title` field is marked as `required=False`, which means it is optional during serialization and deserialization. Ensure this aligns with your application's requirements, as it may allow creating or updating a `Movie` without a title.